va/va.h

```diff
@@ -3625,6 +3625,24 @@ VAStatus vaQuerySurfaceError(
     void **error_info
 );
 
+/**out put value types, used by \vaQueryOutputValue */
+typedef enum
+{
+    VAOutputDecodeCRC = 1
+}VAOutputValueTypes;
+
+/**
+ * this interface is to get some values from driver or HW
+ * after calling vaSyncSurface, the value type is defined with \c VAOutputValueTypes
+ * the output_value should be a pointer allocated by application.
+ * it can be a single value or a structure.
+ */
+VAStatus vaQueryOutputValue(
+    VADisplay dpy,
+    VAContextID context,
```

Comment:
could also add the surfaceID 'VASurfaceID render_target' in this interface? it maybe more flexiable for usage.

Reply:
@Jexu, if add VASurfaceID render_target, it seems that it was changed to another function, calculate the input surface CRC?